guides/v2.2/cloud/project/log-locations.md

```diff
@@ -0,0 +1,106 @@
+---
+group: cloud-guide
+title: Log locations
+functional_areas:
+  - Cloud
+  - Configuration
+redirect_from:
+  - /guides/v2.2/cloud/trouble/environments-logs.html
+  - /guides/v2.3/cloud/trouble/environments-logs.html
+---
+The `/var/log` directory contains logs for all environments. The following logs have a common location for all Cloud projects:
+
+-  **Build log**: `var/log/cloud.log`
+-  **Debug log**: `var/log/debug.log`
+-  **Exception log**: `var/log/exception.log`
+-  **Support report log**: `var/log/support_report.log`
```

Comment:
This seems confusing since the paragraphs around it are discussing `/var/log` which is the system logs, but this block is referring to `var/log/*` which are logs specific to Magento & ECE Tools